dont_clone_monorepo.py

```diff
@@ -0,0 +1,12 @@
+from talon import app
```

Comment:
```suggestion
"""
This file exists in our monorepo so that if someone accidentally clones this monorepo
into their talon folder instead of cursorless-talon, they'll get a helpful error popup
"""
from talon import app
```